README.md

```diff
@@ -32,17 +32,42 @@ We support a growing list of platforms across the desktop, mobile, and console s
 * Mobile/Tablet Devices
   * Android 6.0 and up (OpenGL)
   * iPhone/iPad 10.0 and up (OpenGL)
-* Consoles (for registered developers)
+* [Consoles (for registered developers)](https://docs.monogame.net/articles/console_access.html)
   * PlayStation 4
   * PlayStation 5
   * Xbox One (XDK only) (GDK coming soon)
   * Nintendo Switch
 
+## Getting started
+- [Getting started →](https://docs.monogame.net/articles/getting_started/index.html)
+- ["How To" Guides →](https://docs.monogame.net/articles/getting_to_know/howto/)
+- [Documentation Hub →](https://docs.monogame.net/)
+- [API Reference →](https://docs.monogame.net/api/index.html)
+- [Community Tutorials →](https://docs.monogame.net/articles/tutorials.html)
+
+## [Samples](https://github.com/MonoGame/MonoGame.Samples)
+
+Check out the awesome ["full game" samples](https://github.com/MonoGame/MonoGame.Samples) maintained by the MonoGame team:
+
+[Platformer 2D Sample](https://github.com/MonoGame/MonoGame.Samples/blob/3.8.2/Platformer2D/README.md) | [NeonShooter](https://github.com/MonoGame/MonoGame.Samples/blob/3.8.2/NeonShooter/README.md)|
+|-|-|
+Supported on all platforms | Supported on all platforms |
+[![Platformer 2D Sample](https://raw.githubusercontent.com/MonoGame/MonoGame.Samples/refs/heads/3.8.2/Images/Platformer2D-Sample.png)](https://github.com/MonoGame/MonoGame.Samples/blob/3.8.2/Platformer2D/README.md) | [![NeonShooter Sample](https://raw.githubusercontent.com/MonoGame/MonoGame.Samples/refs/heads/3.8.2/Images/NeonShooter-Sample.png)](https://github.com/MonoGame/MonoGame.Samples/blob/3.8.2/NeonShooter/README.md) |
+The [Platformer 2D](https://github.com/MonoGame/MonoGame.Samples/blob/3.8.2/Platformer2D/README.md) sample is a basic 2D platformer pulled from the original XNA samples and upgraded for MonoGame.| [Neon Shooter](https://github.com/MonoGame/MonoGame.Samples/blob/3.8.2/NeonShooter/README.md) Is a graphically intensive twin-stick shooter with particle effects and save data from Michael Hoffman |
+|||
+
+| [Auto Pong Sample](https://github.com/MonoGame/MonoGame.Samples/blob/3.8.2/AutoPong/README.md) | [Ship Game 3D](https://github.com/MonoGame/MonoGame.Samples/blob/3.8.2/ShipGame/README.md) |
+|-|-|
+| Supported on all platforms | GL / DX/ iOS / Android |
```

Comment:
How about `Desktop and mobile` instead of jargon like `GL / DX / iOS / Android`?

Reply:
Isn't `Desktop and Mobile` already `All platforms`?

Reply:
No...  Desktop, mobile, and console would be all platforms.